Extensions/TileMap/tilemapruntimeobject-pixi-renderer.js

```diff
@@ -43,6 +43,9 @@ gdjs.TileMapRuntimeObjectPixiRenderer.prototype.updateTileMap = function() {
     .getImageManager()
     .getPIXITexture(this._object._tilemapAtlasImage);
   
+  if (!this._object._tilemapAtlasImage) {
+    return
+  };
```

Comment:
Let's run Prettier on all your files to autoformat and remove/add semicolons at the right places.